lib/MwbExporter/Formatter/Doctrine2/Annotation/Model/Column.php

```diff
@@ -33,6 +33,20 @@
 
 class Column extends BaseColumn
 {
+    private function getStringDefaultValue() {
+        $defaultValue = $this->getDefaultValue();
+        if (is_null($defaultValue)) {
+            $defaultValue = '';
+        } else {
+            if ($this->getColumnType() == 'com.mysql.rdbms.mysql.datatype.varchar') {
```

Comment:
Change this IF with the code below:

            if ($this->isStringType()) {
                $defaultValue = " = '$defaultValue'";
            } else {
                $defaultValue = " = $defaultValue";
            }

And add a private method as the following:
    /**
     * Get if the type is a string or not.
     * @return Return true if the datatype is a string, else return false.
     */
    private function isStringType() {
        switch($this->getColumnType())
        {
            case DatatypeConverterInterface::DATATYPE_CHAR:
            case DatatypeConverterInterface::DATATYPE_VARCHAR:
                return true;
            default:
                return false;
        }
    }

And to finish, import the interface:
use MwbExporter\Formatter\DatatypeConverterInterface;